utils/cookiesManager.ts

```diff
@@ -0,0 +1,18 @@
+import { cookies } from "next/headers";
+
+type Props = {
+  name: string;
+  values: any;
```

Comment:
Why is this any here?

Reply:
Or we could use Record type here
```suggestion
  values: Record<string, any>
```